Makefile

```diff
@@ -491,9 +491,12 @@ fmt format: $(GOFUMPT) ## run gofumpt on $FMT_PKG (default "cni cns npm")
 	$(GOFUMPT) -s -w $(FMT_PKG)
 
 # run all tests
+# COVER_PKG omits folders with all files tagged with one of 'unit', '!ignore_test', '!ignore_cli', '!ignore_crd', or '!ignore_autogenerated'
 .PHONY: test-all
 test-all:
-	go test -tags "unit" -coverpkg=./... -v -race -covermode atomic -failfast -coverprofile=coverage.out ./...
+	@$(eval COVER_PKG=`go list --tags ignore_test,ignore_cli,ignore_crd,ignore_autogenerated ./... | tr '\n' ','`)
```

Comment:
i know we're stuck with `ignore_autogenerated` for the controller-gen files, but should we just consolidate on like `uncovered` or something similar for files that are editable?